crud/src/main/java/com/redhat/lightblue/crud/CRUDFindRequest.java

```diff
@@ -187,9 +187,12 @@ public void fromJson(ObjectNode node) {
                 x=node.get("maxResults");
                 if(x!=null) {
                     long l=x.asLong();
-                    if(l>=0) {
+                    if (l > 0) {
                         to=(from==null?0:from)+l-1;
```

Comment:
@bserdar - I have done some testing and gotten some mixed results. Maybe it might be better to get some clarification.

Assuming multiple documents were already inserted. A find operation with (from:0,to:0) should return 1 jsonDoc result or 0 results? If 1 result, then what is the appropriate 'to' value to set for 0 JsonDoc results returned? -1?